spec/models/gestionnaire_spec.rb

```diff
@@ -64,6 +64,52 @@
     end
   end
 
+  describe "#assign_to_procedure" do
+    subject { gestionnaire.assign_to_procedure(procedure_to_assign) }
+
+    context "with a procedure not already assigned" do
+      let(:procedure_to_assign) { procedure_3 }
+
+      it { is_expected.to be_truthy }
```

Comment:
Je chipotte mais je rajouterai bien
`it { expect{ subject }.to change(gestionnaire.procedures, :count) }.by(1)`